### ISSUE TITLE
require("uxp").storage.localFileSystem is broken type

### ISSUE THREAD

**Issue by jardicc** [adobe/cc-ext-uxp-types#2]
`require("uxp").storage.localFileSystem` is broken type

![image](https://github.com/adobe/cc-ext-uxp-types/assets/13526959/300789c7-4568-4955-9d0a-5b36340388a3)

imported like this: `import {storage} from "uxp";` and then `storage.localFileSystem;`

**Comment by andylarquy** [comment]
I patched that type by creating a global-uxp.d.ts file with the following content:
```
export * from 'uxp';

declare module 'uxp' {
  namespace storage {
    var localFileSystem: FileSystemProvider;
  }
}
```

I'm not sure why this wasn't added yet

**Comment by MaxJohnson** [comment]
This continues to be an issue.

**Comment by rrrrigel** [comment]
i just wrote my own instead 😅

**Comment by Jolg42** [comment]
I'm using a fork that has the fix as a temporary solution: https://github.com/justintaylor-dev/cc-ext-uxp-types

**Comment by MaxJohnson** [comment]
> I'm using a fork that has the fix as a temporary solution: https://github.com/justintaylor-dev/cc-ext-uxp-types
Thank you for the link!
I wrote a patch to fix it, but... it really should be fixed on main, right?

**Comment by Jolg42** [comment]
@MaxJohnson yes, the latest commit is about fixing "localFileSystem" https://github.com/justintaylor-dev/cc-ext-uxp-types/commit/4d5d1d2b9b33729a5de44c63aba202efa575d88d

I found it through Bolt UXP "A lightning-fast boilerplate for building Adobe UXP Plugins in Svelte, React, or Vue built on Vite + TypeScript + Sass"

https://github.com/hyperbrew/bolt-uxp/blob/0e83abeca7d50244dd9e5445e7150022f846c0ad/package.json#L40
